packages/lambda-tiler/src/routes/import.ts

```diff
@@ -26,15 +28,24 @@ export async function Import(req: LambdaHttpRequest): Promise<LambdaHttpResponse
     targetTms = tileMatrix;
   }
 
-  // Find the imagery from s3
+  // Validate the s3 path
```

Comment:
@blacha I added some more here, to validate the imagery name before pass it to the CogJobContext.

Reply:
this is going to give very weird imagery names, maybe we should just use `:ulid_:year`  to show its a completly made up name?

Some of these import paths will be completely meaningless, imagine a folder  `s3://a/2022/04/eca/RGB/` what sort of "name" would we get from this?

Reply:
Updated.

Reply:
Wait, we cannot just add uild_year into `CogJobContext`. This will make a different hash each time from `CogJobContext`.

Reply:
I am going to make this imageryName as optional attributions for CogJobContext. And add this after creating hash.